doc/external-access.md

```diff
@@ -115,3 +115,41 @@ example-pravega-segmentstore-headless   ClusterIP      None            <none>
 ```
 
 In the example above, clients will connect to the Pravega Controller at `tcp://35.239.48.145:9090`.
+
+# Using External DNS names for segment store pods
+
+Given a domain suffix for the cluster, each segment store pod can be assigned a DNS name. An external DNS service can resolve this DNS name to the pod externalIP.
+Domain name can be provided in the manifest using `domainname` key under `externalAccess`.
+Segment store dns name is created by prefixing the domain name with segmentstore pod-name.
```

Comment:
Some minor issues. The use of word `segmentstore` is not consistent in this doc, some with space in between and some are not. I think we probably should use `segmentstore` without a space.

Reply:
Sure will fix these..

Reply:
Done.